smart_card_connector_app/src/window-devices-displaying.js

```diff
@@ -83,24 +85,25 @@ function displayReaderList(readers) {
   for (let reader of readers) {
     GSC.Logging.checkWithLogger(logger, readersListElement !== null);
     goog.asserts.assert(readersListElement);
-
+    
     var indicatorClasses = 'reader-state-indicator reader-state-indicator-' +
-                           reader['status'];
+    reader['status'];
     if (reader['status'] == GSC.PcscLiteServer.ReaderStatus.SUCCESS &&
-        reader['isCardPresent']) {
+    reader['isCardPresent']) {
       indicatorClasses = 'reader-card-present-indicator';
     }
     var indicator = goog.dom.createDom('span', indicatorClasses);
-
+    
     var indicatorContainer = goog.dom.createDom(
-        'span', 'reader-indicator-container', indicator);
-
-    var text = makeReaderNameForDisplaying(reader['name']) +
-        (reader['error'] ? ' (Error ' + reader['error'] + ')' : '');
-
-    var element = goog.dom.createDom('li', undefined, indicatorContainer, text);
-    goog.dom.append(readersListElement, element);
-  }
+      'span', 'reader-indicator-container', indicator);
```

Comment:
nit: If I counted correctly, there should be at least two more spaces in front here.

Reply:
There is an issue with the spacing yeah, but I think it's the opposite, that lines 101-103 should have 2 spaces less.